frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-design/aliases-table/aliases-table.component.html

```diff
@@ -90,7 +90,7 @@
     <!-- Actions Column -->
     <ng-container matColumnDef="actions">
       <mat-header-cell *matHeaderCellDef class="ft-14-700"></mat-header-cell>
-      <mat-cell *matCellDef="let rowData; let rowIndex = index" class="actions-cell">
+      <mat-cell *matCellDef="let rowData; let rowIndex = index" class="actions-cell" style="justify-content: flex-start;">
```

Comment:
can these styles be added to a class in scss file instead of inline?

Reply:
yeah I think there's a lot of that inline styling for certain things, I think it's just due to lack of styling guidelines for the project.